README.md

```diff
@@ -1,11 +1,21 @@
-# emcpy
-EMC python tools and utilities
+# EMCPy
+EMCPy is a Python toolkit that supplies useful tools and utilities to help developers at EMC accomplish their Python needs without having to be an expert in Python. It is designed so that novice users can develop plots or calculate statistics with only a few lines of code while giving experienced Python users the ability to be power users. EMCPy utilizes modern program practices  and its centralized functions include plotting routines, statistical calculations, look up tables and helpful I/O procedures.
 
-### Installation
+### Clone Repository
```

Comment:
I think this should be reworded to keep the `pip install` stuff, but mention how this is for machines that we do not "support"